src/parser.cpp

```diff
@@ -415,7 +429,8 @@ Variant* VariantParser<DoCheck>::parse(Variant* v) {
                 std::cerr << "Can not use kings with blastOnCapture." << std::endl;
             if (v->flipEnclosedPieces)
                 std::cerr << "Can not use kings with flipEnclosedPieces." << std::endl;
-            // We can not fully check custom king movements at this point
+            // We can not fully check support for custom king movements at this point,
+            // since custom pieces are only initialized on loading of the variant.
```

Comment:
Maybe add something like "We will assume this is valid, but it might cause problems later if it's not". That would clear it up for me